src/vcpkg/binarycaching.cpp

```diff
@@ -1398,7 +1452,7 @@ namespace vcpkg
                                      }
                                      else
                                      {
-                                         print2("unknown key: ", key, '\n');
+                                         msg::println(msgUnknownKey, msg::value = key);
```

Comment:
```suggestion
                                         Debug::println("unknown key: ", key);
```

Since this is just before `unreachable` I don't think it's intended to ever be printed but printing it as debug text may make sense.

Reply:
Since this is intended to be a diagnostic for programmer errors, perhaps it makes sense to have a "Debug print always" function that's unlocalized and only intended to be used for hard errors like before an "unreachable"?

Reply:
That seems reasonable to me